packages/client/components/EndMeetingButton.tsx

```diff
@@ -14,9 +15,8 @@ import {ElementWidth, Times} from '../types/constEnums'
 import isDemoRoute from '../utils/isDemoRoute'
 import BottomNavControl from './BottomNavControl'
 import BottomNavIconLabel from './BottomNavIconLabel'
-import Icon from './Icon'
 
-const FlagIcon = styled(Icon)({
+const FlagIcon = styled(Flag)({
   color: PALETTE.SKY_500,
   height: 24
```

Comment:
+1 24 is the default height for MUI icons

Reply:
👍